### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by elenapeskova**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -1,28 +1,132 @@
...
-* [Default3.aspx.cs](./CS/Default3.aspx.cs) (VB: [Default3.aspx.vb](./VB/Default3.aspx.vb))
-<!-- default file list end -->
-# How to show ASPxPopupControl on the ASPxGridView's CustomButton click
+# Grid View for ASP.NET Web Forms - Show the Popup Control on the Grid's Custom Button Click
+
 <!-- run online -->
 **[[Run Online]](https://codecentral.devexpress.com/e4424/)**
 <!-- run online end -->
 
+This example demonstrates how to show the [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) on the [ASPxGridView](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxGridView) custom button click.
```
Comment (dirk-pieterse) [inline Readme.md]: Can you please clarify what a `custom button click` is?

**Review by dirk-pieterse — APPROVED** [review]

File: Readme.md (hunk 1 above)
Comment (elenapeskova) [reply Readme.md]: There is a custom button in the grid's Command column. When a user clicks this button, the popup control is shown.

Comment (dirk-pieterse) [reply Readme.md]: Then something like `This example demonstrates how to show the [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) when a user clicks the custom button in the grid's Command column` would be better

Comment (elenapeskova) [reply Readme.md]:
```suggestion
This example demonstrates how to show the [ASPxPopupControl](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPopupControl) when a user clicks the custom button in the grid's command column.
```

**Pull request merged**